frontend/src/components/InputAndControlsPanel.tsx

```diff
@@ -180,6 +186,9 @@ export function InputAndControlsPanel({
               <ToggleGroupItem value="camera" aria-label="Camera">
                 Camera
               </ToggleGroupItem>
+              <ToggleGroupItem value="spout" aria-label="Spout">
```

Comment:
```suggestion
              <ToggleGroupItem value="spout" aria-label="Spout Receiver">
```

if we just call the source type this then we could reduce the amount of visual space we need to take up below.